CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -32,6 +32,9 @@
 == 2021-03-22 Scatter Plot
 <p>Ctrl-Shift-O has been added as a keyboard shortcut to open up the Scatter Plot View.</p>
 
+== 2021-02-25 Conversation View
```

Comment:
Given it's been a while since this was first submitted, could be worth updating the date here and moving it up so it can be seen by users